cognite/neat/_utils/spreadsheet.py

```diff
@@ -148,10 +148,10 @@ def find_column_with_value(sheet: Worksheet, value: Any) -> str | None:
     return None
 
 
-def generate_data_validation(sheet: str, column: str, no_header_rows: int, no_rows: int) -> DataValidation:
+def generate_data_validation(sheet: str, column: str, no_header_rows: int, no_validation_values: int) -> DataValidation:
```

Comment:
I read this as **no** validation values and expects a bool.
```suggestion
def generate_data_validation(sheet: str, column: str, no_header_rows: int, validation_value_count: int) -> DataValidation:
```

Reply:
renamed to total_validation_values